Sources/GRPC/ConnectionPool/GRPCChannelPool.swift

```diff
@@ -275,6 +275,10 @@ extension GRPCChannelPool.Configuration {
     /// Defaults to 100.
     public var maxWaitersPerEventLoop: Int = 100
 
+    /// The minimum number of connections to keep open in this pool.
+    /// This number of connections will never go idle and be closed.
+    public var minConnections: Int = 1
```

Comment:
As implemented this is actually min connections per event loop (which is probably what we want, but we should make that more obvious.)

Reply:
Sorry, I meant by making it more explicit in the name (i.e. like `connectionsPerEventLoop` and `maxWaitersPerEventLoop`)